functorch/_src/eager_transforms.py

```diff
@@ -17,28 +17,24 @@
     _grad_decrement_nesting,
 )
 
-# TODO: replace all of these with pytrees
-def _create_differentiable(tensor_or_tuple_of_tensors, level=None):
-    if isinstance(tensor_or_tuple_of_tensors, torch.Tensor):
-        tensor = tensor_or_tuple_of_tensors
-        aliased = tensor
-        return aliased.requires_grad_()
-    if isinstance(tensor_or_tuple_of_tensors, tuple):
-        return tuple(map(partial(_create_differentiable, level=level), tensor_or_tuple_of_tensors))
-    if isinstance(tensor_or_tuple_of_tensors, list):
-        return tuple(map(partial(_create_differentiable, level=level), tensor_or_tuple_of_tensors))
-    raise ValueError(f'Thing passed to transform API must be Tensor, List or Tuple, '
-                     f'got {type(tensor_or_tuple_of_tensors)}')
-
-def _undo_create_differentiable(tensor_or_tuple_of_tensors, level=None):
-    if isinstance(tensor_or_tuple_of_tensors, torch.Tensor):
-        tensor = tensor_or_tuple_of_tensors
-        return _unwrap_for_grad(tensor, level)
-    if isinstance(tensor_or_tuple_of_tensors, tuple):
-        return tuple(map(partial(_undo_create_differentiable, level=level), tensor_or_tuple_of_tensors))
-    if isinstance(tensor_or_tuple_of_tensors, list):
-        return tuple(map(partial(_undo_create_differentiable, level=level), tensor_or_tuple_of_tensors))
-    assert False
+def _create_differentiable(inps, level=None):
+    def create_differentiable(x):
+        if isinstance(x, torch.Tensor):
+            tensor = x
+            aliased = tensor
+            return aliased.requires_grad_()
```

Comment:
nit: `return x.requires_grad_()`. I wrote the code this way originally because it used to do a bit more computation rather than just reassign variables